lib/mri/forwardable.rb

```diff
@@ -187,9 +184,6 @@ def def_instance_delegator(accessor, method, ali = method)
       def #{ali}(*args, &block)
         begin
           #{accessor}.__send__(:#{method}, *args, &block)
-        rescue ::Exception
-          $@.delete_if{|s| ::Forwardable::FILE_REGEXP =~ s} unless ::Forwardable::debug
-          ::Kernel::raise
```

Comment:
What's problematic here?

Reply:
I don't think anything's problematic - what do you mean?

Reply:
I mean, why did you remove those lines? There were modified previously?

Reply:
Oh the equivalent rescues in the MRI implementation were deleted by them - the semantics changed.